osm2gtfs/creators/ci_abidjan/README.md

```diff
@@ -0,0 +1,14 @@
+# Abidjan config for osm2gtfs
+
+This config for osm2gtfs is used to produce the GTFS for Abidjan, Ivory Coast.
+
+It was created in 2019 for the Ivory Coast Ministry of Transport, as part of a project financed by 
+l'Agence Française de Développement (AFD) and carried out by Jungle Bus, Systra and the local 
+OpenStreetMap Ivory Coast association (OSM-CI).
+
+
+Useful resources:
+* [osm2gtfs config](./config.json)
+* [Documentation related to the project](https://git.digitaltransport4africa.org/data/africa/abidjan)
+* [osm2gtfs user manual for Accra (TODO: create user manual for Abidjan)](https://wiki.openstreetmap.org/wiki/OSM2GTFS_for_Accra_-_user_manual)
```

Comment:
No TODO please, we will add the tuto here if / when we write it

Reply:
ok